### PR TITLE
Use sitemap from appium instead of hardcoding     

### PR DESCRIPTION
**Pull request by dpgraham**
* Pulls sitemap.json from the docs directory in Appium instead of hardcoding it into repo.js
* Also, strip out python style comments from code examples

### CODE REVIEW

**Review by imurchie — APPROVED** [review]
Excellent! Much nicer to have it there.

**Review by jlipps — APPROVED** [review]
looks good. guess we should merge once the other appium PR is merged

File: scripts/repo.js [hunk 1]
```diff
@@ -182,11 +118,13 @@ function buildDocYML (sitemap, baseDir='', levelCount=0) {
 async function buildDocs (pathToDocs) {
   const mkdocsTemplate = Handlebars.compile(await fs.readFile(path.resolve(__dirname, '..', 'mkdocs.yml'),  'utf8'));
   const themeDir = path.resolve(__dirname, '..', 'cinder');
+  const sitemapJSON =  await fs.readFile(path.resolve(pathToDocs, 'toc.json'), 'utf8');
```
Comment (mykola-mokhnach) [inline scripts/repo.js]: I thought this returns a Buffer rather than a string

Comment (dpgraham) [reply scripts/repo.js]: Adding 'utf8' as the second argument turns it into a string.

Comment (jlipps) [reply scripts/repo.js]: you could just `require` it. then it doesn't matter whether it's js or json, and it turns it into an obj

**Comment by dpgraham** [comment]
Changed it so that it requires `toc.js` instead of parsing `toc.json`

**Pull request merged**